packages/reason-css-parser/lib/Parser.re

```diff
@@ -1807,14 +1807,16 @@ and symbol = [%value.rec "<string> | <image> | <custom-ident>"]
 and target = [%value.rec
   "<target-counter()> | <target-counters()> | <target-text()>"
 ]
-and extended_length = [%value.rec "<length> | <calc()> | <interpolation>"]
+and extended_length = [%value.rec
+  "<length> | <calc()> | <interpolation> | <min()> | <max()>"
+]
 and extended_frequency = [%value.rec
-  "<frequency> | <calc()> | <interpolation>"
+  "<frequency> | <calc()> | <interpolation> | <min()> | <max()>"
 ]
 and extended_angle = [%value.rec "<angle> | <calc()> | <interpolation>"]
 and extended_time = [%value.rec "<time> | <calc()> | <interpolation>"]
```

Comment:
time and angle support min/max as well https://developer.mozilla.org/en-US/docs/Web/CSS/max